alts/src/main/java/io/grpc/alts/internal/AltsProtocolNegotiator.java

```diff
@@ -119,7 +122,8 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
                     .set(TSI_PEER_KEY, altsEvt.peer())
                     .set(ALTS_CONTEXT_KEY, altsContext)
                     .set(Grpc.TRANSPORT_ATTR_REMOTE_ADDR, ctx.channel().remoteAddress())
-                    .build());
+                    .build(),
+                new Security(new OtherSecurity("alts", Any.pack(altsContext.context))));
```

Comment:
FYI: `Any.pack()` eagerly serializes the content. I'm not too concerned; AltsContext is small and this is only done after we've done after a handshake.

Reply:
Right, AltsContext is smaller than the TLS info.